docs/building-apps/wallet/sep24.mdx

```diff
@@ -143,6 +189,18 @@ suspend fun depositDifferentAccount(): InteractiveFlowResponse {
 }
 ```
 
+```typescript
+const recipientAccount = "G...";
+const depositDifferentAccount = async (): Promise<InteractiveFlowResponse> => {
+  return anchor.interactive().deposit({
+    accountAddress: accountKp.publicKey(),
+    assetCode,
+    authToken,
+    fundsAccountAddress: recipientAccount
```

Comment:
I believe `fundsAccountAddress` is the equivalent key for `destinationAccount` above, but lmk if I'm wrong

Reply:
I think so. It should be optional, is it like this right now? (If not, making it optional is part of WAL-889 IIRC)

Reply:
`fundsAccountAddress` is optional. It gets set to `accountAddress` if it isn't explicitly set and then sent to the `/interactive` endpoint as `account`

Reply:
Got it. Small thing: for withdrawal it should be `originAccount`